referenceframe/model_json_test.go

```diff
@@ -34,8 +34,8 @@ func TestParseJSONFile(t *testing.T) {
 		ErrCircularReference,
 		NewReservedWordError("link", "world"),
 		NewReservedWordError("joint", "world"),
-		ErrAtLeastOneEndEffector,
-		NewFrameNotInListOfTransformsError("base"),
```

Comment:
This test case was testing that a base frame with parent 'world' didn't exist.  In solving this ticket this is no longer valid so now I am testing that if a frame is missing that it fails correclly (it would look like 2 end effectors)

The corresponding file is also changed below

Reply:
Just confirming: By "if a frame is missing" do you mean a disconnected graph or that it is missing the desired 'base/world frame'